output/schema.json

```diff
@@ -46,18 +46,18 @@
         },
         {
           "if": {
-            "oneOf": [
+            "anyOf": [
```

Comment:
Why this change?  Are you expecting `$ref` and `$recursiveRef` to both appear in the keyword location?

... I suppose they could....

okay.

Reply:
sure, if there is a $recursiveRef in the subschema at the other end of a $ref, it could certainly happen.

I have a test case with `.../$ref/$ref/$ref/$ref/..` :) -- https://github.com/karenetheridge/JSON-Schema-Draft201909/blob/master/t/errors.t#L788